microcloud/test/includes/microcloud.sh

```diff
@@ -959,9 +959,9 @@ setup_system() {
 
     if [ -n "${MICROCLOUD_SNAP_PATH}" ]; then
       lxc file push "${MICROCLOUD_SNAP_PATH}" "${name}"/root/microcloud.snap
-      lxc exec "${name}" -- sh -c "PATH=\$PATH:/snap/bin snap install --devmode /root/microcloud.snap"
+      lxc exec "${name}" -- snap install --devmode /root/microcloud.snap
```

Comment:
where does /root/microcloud.snap come from? Should we be sideloading the PR's binary into the edge microcloud snap instead?

Reply:
We should be using `--dangerous` rather than `--devmode` as the latter isn't confined and isn't a representative test.